popup/popup.js

```diff
@@ -576,10 +577,18 @@ Object.assign(handleEvent, {
             .then(() => msg.sendTab(tab.id, message));
         }
       })
-      .then(window.close);
+      .then(() => {
+        // Chromium needs help closing popup when opening new windows, not same window tabs
+        // FF needs help closing popup when opening same window tabs, not new windows
+        // edit URLs are ignored by pref - manager URls are passed to openURL() to check if tab exists
+        // which sends a message to close popup after determining it's not a different FF window
+        // because closing popup interferes with activating different windows in FF
+        if (this.dataset.href !== 'manage.html') window.close();
```

Comment:
Why do we want to keep the window when `dataset.href` is `manage.html`? Does `window.close` activate the current window?

Reply:
Same as I've answered multiple times. Avoid the FF bug when activating tabs in other windows.